client/src/components/CounselorDashboard/Dashboard.js

```diff
@@ -0,0 +1,16 @@
+import React, { useState } from 'react'
+import AddPatient from './AddPatient'
+import PatientList from './PatientList'
+
+function Dashboard({ addPatient, patients }) {
+  const [add, setAdd] = useState(false);
+
+  return (
+    <div>
+      {add ? <button onClick={() => setAdd(false)}>Back</button> : <button onClick={() => setAdd(true)}>Add Patient</button>}
```

Comment:
ill clean this up as i add the styling in the next pr

Reply:
sounds good